docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -20,35 +17,37 @@
           flat
           icon="menu"
           round
-          @click="$emit('update:modelValue', !modelValue)"/>
+          @click="$emit('update:modelValue', !modelValue)"
+        />
         <router-link
           v-if="$q.screen.gt.xs || !isSearchFieldActive"
           class="row justify-center items-center cursor-pointer"
           :to="{ name: 'home' }"
         >
           <img
             v-if="$q.screen.sm"
-            :src="`https://cdn.quasar.dev/logo-v2/svg/logo${!dark? '-dark':''}.svg`"
+            :src="`https://cdn.quasar.dev/logo-v2/svg/logo${!dark ? '-dark' : ''}.svg`"
             alt="Quasar Logo"
             height="48"
-            width="48">
+            width="48"
+          />
           <img
             v-else
-            :height="$q.screen.xs? '24':'48'"
-            :src="`https://cdn.quasar.dev/logo-v2/svg/logo-horizontal${!dark? '-dark':''}.svg`"
+            :height="$q.screen.xs ? '24' : '48'"
+            :src="`https://cdn.quasar.dev/logo-v2/svg/logo-horizontal${!dark ? '-dark' : ''}.svg`"
             alt="Quasar Logo"
-            width="236">
-          <q-separator
+            width="236"
+          />
+          <!-- <q-separator
```

Comment:
Because it's ugly. 

Scott

Reply:
I think it's out of the scope of adding dark mode. It's a rather subjective opinion, so I think leaving the separator as is(_i.e. not commenting it out_) and letting the time and users decide if that could be useful or useless, beautiful or ugly, would be a good idea.

Reply:
We defined the scope of the change into the meeting, and the team decided to avoid other GUI changes in the previous meeting.
I'm sorry, but your definition of what is ugly and what is not isn't really relevant.
Please re-add it